rpc/common/moc_common_common.proto

```diff
@@ -348,8 +349,9 @@ enum HyperVGeneration {
 } 
 
 enum DiskFileFormat {
-	DiskFileFormatVHDX =  0;
-	DiskFileFormatVHD = 1;
+	DiskFileFormatUNKNOWN = 0;
+	DiskFileFormatVHDX =  1;
+	DiskFileFormatVHD = 2;
```

Comment:
we should only append to the enum. If we change this order and an old object has DiskFileFormat based on the previous values, their meaning will change after the upgrade.

Reply:
Replied about this on teams

Reply:
Done